src/components/AccountDropdown/AccountDropdown.tsx

```diff
@@ -10,24 +11,29 @@ import {
   Text,
 } from '@chakra-ui/react'
 import {
-  AccountId,
-  AssetId,
+  type AccountId,
```

Comment:
is there a lint rule/code mod for this?

Reply:
You betcha https://github.com/shapeshift/web/pull/2691